perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -283,11 +284,31 @@ def _GenerateCreateCommand(self, ssh_keys_path):
       cmd.flags['machine-type'] = self.machine_type
     cmd.flags['tags'] = 'perfkitbenchmarker'
     cmd.flags['no-restart-on-failure'] = True
-    cmd.flags['metadata-from-file'] = 'sshKeys=%s' % ssh_keys_path
-    metadata = ['owner=%s' % FLAGS.owner]
-    for key, value in self.boot_metadata.iteritems():
-      metadata.append('%s=%s' % (key, value))
-    cmd.flags['metadata'] = ','.join(metadata)
+
+    metadata_from_file = {'sshKeys': ssh_keys_path}
+    for pair in FLAGS.gcp_instance_metadata_from_file:
```

Comment:
The flag description says comma-separate key-value pairs are supported, but I think DEFINE_multistring only allows the flag to be repeated and doesn't support the comma behavior by default. The `--metadata` flag had to handle it explicitly, see here: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/publisher.py#L180